### PR TITLE
Add command to print ouptut to console

### PR DESCRIPTION
**Pull request by AlexandreDewilde**
For #38

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def check_pre_requisites_tesseract():
         return True
 
 
-def main(input_path, output_path):
+def main(input_path, output_path, no_output_directory=False):
```
Comment (prabhakar267) [inline main.py]: Why do you need a new parameter to detect is output directory was provided or not?
You can pass `output_path` as `None` and use that as your flag.

**Comment by allEyezOnCode** [comment]
So the line 141 like that?:
`if os.path.join(input_path, 'converted-text') == output_path:`

**Comment by allEyezOnCode** [comment]
Or should I rewrite the run_tesseract function to return a list or a dic in case the output_dir flag = none and print the list ?

**Comment by AlexandreDewilde** [comment]
Or should I rewrite the run_tesseract function to return a tempfile in case the output_dir flag = None?

File: main.py [hunk 2]
```diff
@@ -138,6 +138,9 @@ def main(input_path, output_path):
                 "Successfully parsed images: {}".format(successful_files))
             logging.info(
                 "Files with unsupported file extensions: {}".format(other_files))
+            if no_output_directory:
+                for file in os.listdir('{}/converted-text'.format(input_path)):
```
Comment (prabhakar267) [inline main.py]: You have implemented this for each file instead it should be a global flag i.e., if the output_path was not passed in arguments print output of every file on console and do not save it to any file

File: main.py (hunk 1 above)
Comment (prabhakar267) [inline main.py]: Why do you need a new parameter to detect is output directory was provided or not?
You can pass `output_path` as `None` and use that as your flag.

**Review by prabhakar267 — CHANGES_REQUESTED** [review]

**Comment by AlexandreDewilde** [comment]
So I rewrite the run_tesseract function to return the text in case no output path is provided, I hope it's OK

**Review by prabhakar267 — APPROVED** [review]

**Pull request merged**